src/messages/MessageBuilder.cpp

```diff
@@ -385,8 +385,26 @@ EmotePtr makeAutoModBadge()
         Url{"https://dashboard.twitch.tv/settings/moderation/automod"}});
 }
 
-EmotePtr makeSharedChatBadge(const QString &sourceName)
+EmotePtr makeSharedChatBadge(const QString &sourceName, const QString &url)
 {
+    if (!url.isEmpty())
+    {
+        QString modifiedUrl = url;
+
+        modifiedUrl.replace("300x300", "70x70");
+
+        return std::make_shared<Emote>(Emote{
+            .name = EmoteName{},
+            .images = ImageSet{Image::fromUrl(
+                Url{modifiedUrl},
+                0.257142857143)},  // get as close to 18x18 as possible
```

Comment:
```suggestion
                18.F/28.F)},  // get as close to 18x18 as possible
```

(or whatever the desired size is)

Reply:
Fixed in [`d99a439` (#5760)](https://github.com/Chatterino/chatterino2/pull/5760/commits/d99a439945097ae2b8aaaa0ddc48547066c2648b)